index.html

```diff
@@ -33,11 +33,179 @@ <h2 id="id-my-name">Marc Raphael</h2>
+        <ul id="id-project-container">
+          
+          <!--project one-->
+          <li class="class-project">
+            <img src="./images/Rectangle21.png" alt="project image" class="class-project-image">
+            <div class="class-project-description">
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+          
+          <!--project two-->
+          <li class="class-project">
+            <img src="./images/Rectangle21(2).png" alt="project image" class="class-project-image">
+            <div class="class-project-description">            
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+
+          <!--project three-->
+          <li class="class-project">
+            <img src="./images/Rectangle21(3).png" alt="project image" class="class-project-image">
+            <div class="class-project-description">
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
+                <li class="class-project-li">HTML/CSS</li>
+                <li class="class-project-li">Ruby on rails</li>
+                <li class="class-project-li">Javascript</li>
+              </ul>
+              <a href="#" target="_parent" class="class-project-link">
+                <span class="class-project-button">See this Project</span>
+                <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+              </a>
+            </div>
+          </li>
+
+          <!--project four-->
+          <li class="class-project">
+            <img src="./images/Rectangle21(4).png" alt="project image" class="class-project-image">
+            <div class="class-project-description">            
+              <ul class="class-project-languages">
+                <li class="class-project-name">Project name goes here</li>
```

Comment:
- [ ] You shouldn't wrap this element as a `li` or list element. It should be a heading/title with `hx` tag. Kindly make change on this.